repository-hpi/src/main/java/com/nirima/jenkins/repo/project/ProjectsElement.java

```diff
@@ -47,10 +49,10 @@ public String getName() {
 
     public Collection<ProjectElement> getChildren() {
 
-        List<ProjectElement> elements = Lists.newArrayList(Iterators.transform(Jenkins.getInstance().getAllItems(BuildableItemWithBuildWrappers.class).iterator(),
-                new Function<BuildableItemWithBuildWrappers,ProjectElement>()
+        List<ProjectElement> elements = Lists.newArrayList(Iterators.transform(Jenkins.getInstance().getItems().iterator(),
```

Comment:
This should return only root items. `ProjectElement` has its own `getChildren` method.